pkg/segment/search/searchaggs.go

```diff
@@ -264,13 +272,22 @@ func addRecordToAggregations(grpReq *structs.GroupByRequest, timeHistogram *stru
 				}
 			}
 		} else {
+
+			// resize the working buf if we cannot accomodate the max value of any
+			// column's record
+			if len(aggsKeyWorkingBuf) < len(groupbyColKeyIndices)*65_000 {
+				aggsKeyWorkingBuf = toputils.ResizeSlice(aggsKeyWorkingBuf,
+					len(groupbyColKeyIndices)*65_000)
+			}
```

Comment:
Where is 65,000 coming from? Maybe this should be a constant

Reply:
it is a max value of any column value, switching it to a constant in the next commit